src/front/glsl/variables.rs

```diff
@@ -310,7 +311,7 @@ impl Program<'_> {
             self.entry_args.push((
                 Binding::Location {
                     location,
-                    interpolation,
+                    interpolation: Some(interpolation.unwrap_or(Interpolation::Perspective)),
```

Comment:
careful with the type here: it can only be Perspective for floating point stuff, has to be Flat for integers

Reply:
Read on the glsl spec that the default was perspective but missed that it was only for floats